02week/bio/css/style.css

```diff
@@ -0,0 +1,37 @@
+body{
+  background-color: pink;
+  color: white;
+}
+
+h1{
+  color: black;
+}
+
+h4{
+  color: lightblue;
+}
+
+.First{
+  border: 5px dashed black;
+  background-color: grey;
+  margin: 20px;
+  padding-left: 10px;
+  width: 500px;
+}
+
+.Second{
+  border: 5px dotted black;
+  background-color: grey;
+  margin-left: 450px;
+  padding-left: 10px;
+  width: 500px;
+}
+
+.Third{
+  border: 5px solid black;
+  background-color: grey;
+  margin-left: 900px;
+  margin-top: 20px;
+  padding-left: 10px;
+  width: 500px;
+}
```

Comment:
Styling looks awesome!! I would recommend using things other than margins to make your boxes look like that across the page, because it hides them if you make the browser window smaller. We will learn more ways to do this, but it works for right now!